x-pack/qa/oidc-op-tests/src/test/java/org/elasticsearch/xpack/security/authc/oidc/OpenIdConnectAuthIT.java

```diff
@@ -420,6 +420,7 @@ private PrepareAuthResponse getRedirectedFromFacilitator(String realmName) throw
             logger.info(" OpenIDConnect authentication response {}", responseBody);
             assertNotNull(responseBody.get("access_token"));
             assertNotNull(responseBody.get("refresh_token"));
+            assertNotNull(responseBody.get("realm_name"));
```

Comment:
A similar assertion for `realm_name` of the prepare response can also be added in method `OpenIdConnectAuthIT#getRedirectedFromFacilitator`.

Reply:
Made realm param no nullable for PrepareAuthResponse called from OpenIdConnectAuthIT#getRedirectedFromFacilitator